schemas/database/004_identities.up.sql

```diff
@@ -0,0 +1,14 @@
+-- Add the new 'id' column
+ALTER TABLE identities ADD COLUMN id BIGSERIAL;
```

Comment:
We should also make sure that the initial value of `id` is the same as `leaf_index`